### PR TITLE
Upgrade @polkadot/api for parachain types

### PR DESCRIPTION
**Pull request by vgeddes**
The API types for Rococo parachains are now bundled in the latest `@polkadot/api`. See [hrmp.ts](https://github.com/polkadot-js/api/blob/01098395ec65aff67373e01de9421746eb723e98/packages/types/src/interfaces/parachains/hrmp.ts) for example.

Have tested a parachain launch successfully after the upgrade. The parachain collator produced blocks successfully.

Also updated the README to reflect the de-facto installation method for the tool, as the existing steps will result in users getting an ancient incompatible version installed from NPM.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -51,274 +51,5 @@
...
-				"Implicit": "ValidatorSignature",
-				"Explicit": "ValidatorSignature"
-			}
-		},
-		"BackedCandidate": {
-			"candidate": "CommittedCandidateReceipt",
-			"validity_votes": "Vec<ValidityAttestation>",
-			"validator_indices": "BitVec"
-		},
-		"OriginKind": {
```
Comment (pepyakin) [inline config.json:111]: I am not sure if we can remove all the below. That's the XCM stuff and I assume it is still not present on the polkadot.js side.

File: README.md [hunk 2]
```diff
@@ -3,13 +3,16 @@
...
-Or, if you use `npm`:
 ```bash
-npm i polkadot-launch -g
+git clone https://github.com/paritytech/polkadot-launch
+cd polkadot-launch
+
+yarn install
+yarn build
+
+yarn global add file:$(pwd)
```
Comment (pepyakin) [inline README.md:15]: care to leave an inline comment what this is supposed to do?

File: config.json (hunk 1 above)
Comment (vgeddes) [reply config.json:111]: Its all there: https://github.com/polkadot-js/api/blob/640ef4bdab2a15df8c2a752fe4ba75d37ed813bf/packages/types/src/interfaces/parachains/definitions.ts#L392 🙂

And is being actively maintained: https://github.com/polkadot-js/api/pull/3067

Comment (pepyakin) [reply config.json:111]: fair enough then

**Comment by vgeddes** [comment]
This is now quite out of date and shouldn't be merged, so closing for now. 

To get polkadot-launch working with the very latest rococo-v1, need to run `yarn upgrade @polkadot/api` and also add types for `MultiAddress` to the config (polkadot now uses MultiAddress).

**Pull request closed without merging**